ddtrace/internal/datadog/profiling/stack_v2/CMakeLists.txt

```diff
@@ -0,0 +1,117 @@
+cmake_minimum_required(VERSION 3.19)
+
+# We're building a Python extension and expect to be called from setup.py
+set(EXTENSION_NAME "" CACHE STRING "Name of the extension")
+project(${EXTENSION_NAME})
+
+# Set C++17 as required standard for the whole project.
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -Wextra")
+
+# Features
+option(UNWIND_NATIVE_ENABLE "Enable native unwinding" OFF)
+
+# Includes
+include(FetchContent)
+include(ExternalProject)
+include(FindPython3)
+add_subdirectory(../dd_wrapper "${LIB_INSTALL_DIR}")
+
+# Find the Python interpreter
+find_package(Python3 COMPONENTS Interpreter REQUIRED)
+if (NOT Python3_INCLUDE_DIRS)
+  message(FATAL_ERROR "Python3_INCLUDE_DIRS not found")
+endif()
+
+# Add echion
+FetchContent_Declare(
+    echion
+    GIT_REPOSITORY "https://github.com/sanchda/echion.git"
+    GIT_TAG "sanchda/more_rendering"
+)
+FetchContent_GetProperties(echion)
+if(NOT echion_POPULATED)
+  FetchContent_Populate(echion)
+endif()
+
+# Get libunwind
+if(UNWIND_NATIVE_ENABLE)
```

Comment:
The aim here is to also add the native part of stacks to the python stacks ?
Is the addition of this dependency not something that impact 3rd party files and licenses ?